plugins/package-managers/gleam/src/main/kotlin/Gleam.kt

```diff
@@ -21,6 +21,8 @@ package org.ossreviewtoolkit.plugins.packagemanagers.gleam
 
 import java.io.File
 
+import net.peanuuutz.tomlkt.TomlElement
```

Comment:
Note that this now introduces implementation details of the serializer in use to `Gleam.kt`, which I believe goes a bit against the encapsulation arguments which you have been using before, or?

Reply:
> which I believe goes a bit against the encapsulation arguments which you have been using before

(I guess the arguments you referr to had the intention of having the serialization in one file, e.g. `parse()` next to the model, and not spread in multiple files).

The import here was only used for an explicit return type. I do not see how this breaks encapsulation. Anyhow, 
I dropped the explicit return type along with the import.

Reply:
> I do not see how this breaks encapsulation.

Just to explain: The Gleam implementation should not need to know any details of the concrete TOML deserializer being used. But as soon as anything from `net.peanuuutz.tomlkt` is imported, that's an indication that this constraint is broken.

In essence, a change of the TOML deserialization library should not require any changes to `Gleam.kt`, ideally.

Reply:
That's clear, but the "problem" comes from the Toml data model exposing the `net.peanuuutz.tomlkt` in the first place, not from the code making use of it, or?

Reply:
> Do you feel like making that change? Otherwise I will.

I'll have a look at this.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/11243.